src/main/java/net/minestom/server/MinecraftServer.java

```diff
@@ -478,9 +476,7 @@ public static void setChunkViewDistance(int chunkViewDistance) {
         MinecraftServer.chunkViewDistance = chunkViewDistance;
         if (started) {
 
-            final Collection<Player> players = connectionManager.getOnlinePlayers();
-
-            players.forEach(player -> {
+            for (final Player player : connectionManager.getOnlinePlayers()) {
```

Comment:
Doing this *might* be a bad idea. Take Java's `SynchronizedCollection` for example where `forEach()` is synchronised but the iterator, spliterator, stream, and parallelStream are all explicitly labelled with "Must be manually synched by user!" It might be better to defer to the collection's encapsulation for iteration, particularly when retrieving information outside of an instance's thread. Not sure how much of a problem this is, if any, but just something to keep in mind.

Reply:
You are right but this is not an issue in this case since sending packets is thread-safe, and #refreshVisibleChunks only read. And reading a wrong value there will not mess with anything